### PR TITLE
Applied the note shortcode to the notes

### PR DESCRIPTION
**Pull request by Rajakavitha1**
Applied the note shortcode: https://github.com/mattermost/mattermost-developer-documentation/tree/master/site/layouts/shortcodes
to the content for better format.

### CODE REVIEW

**Comment by jasonblais** [comment]
@Rajakavitha1 @hanzei Do we have an example on what the note looks like visually?

**Comment by hanzei** [comment]
@jasonblais 
Screenshot from 2019-07-21 23-20-52

File: site/content/contribute/getting-started/inactive-contributions.md [hunk 1]
```diff
@@ -13,13 +13,15 @@ This process describes how inactive contributions are managed at Mattermost, ins
...
 
 3. After 30 days of inactivity, a contribution becomes orphaned.
  - Community Coordinator notes that the contribution has been inactive for 60 days, thanks for the contribution and closes the contribution. They also add an `lifecycle/3:orphaned` label to the contribution, and adds an `Up For Grabs` label to the associated help wanted ticket, if appropriate.
 
-Special notes:
-
-1. If the contribution is inactive but shouldn't be closed, the Coordinator adds a `lifecycle/frozen` label to the contribution. An example of this is when a design decision is being discussed but no decision has been arrived at yet.
-2. Once the contribution reaches the `lifecycle/2:inactive` state, it is eligible to be assumed by another community member interested in working on the ticket.
-3. Invalid PRs may be closed immediately without advancing through this lifecycle, especially if the contributor is unresponsive.
+{{< note >}}
```
Comment (hanzei) [inline site/content/contribute/getting-started/inactive-contributions.md]: I'm not fully convinced that this should be wrapped in a note. To me this is more a "normal" section and less an additional note.

File: site/content/contribute/getting-started/inactive-contributions.md [hunk 2]
```diff
@@ -13,13 +13,15 @@ This process describes how inactive contributions are managed at Mattermost, ins
 
 2. After 20 days of inactivity, a contribution becomes inactive.
  - Community Coordinator asks the submitter if the team can help with questions. They acknowledge that after another 30 days of inactivity the contribution will be closed. They also add a `lifecycle/2:inactive` label to the contribution.
- - Note: Contributions should never become orphaned because of reviewers. The Coordinator will be responsible for receiving a response from the reviewers during the stale period, which may be that the maintainers aren't able to accept the contribution in its current form.
+{{< note >}}
```
[the remark this answers is not in the document]
Comment (Rajakavitha1) [reply site/content/contribute/getting-started/inactive-contributions.md]: Do I raise a new PR to update the margin as you suggest. the changes need to be done in https://github.com/mattermost/mattermost-developer-documentation/blob/master/site/layouts/shortcodes/note.html and the CSS in https://github.com/mattermost/mattermost-developer-documentation/blob/master/site/static/css/note.css?

File: site/content/contribute/getting-started/inactive-contributions.md (hunk 1 above)
Comment (Rajakavitha1) [reply site/content/contribute/getting-started/inactive-contributions.md]: Thanks @hanzei !!!! Yes, I agree. Do we rephrase the heading to something that reads: 'Exceptions'

File: site/content/contribute/getting-started/inactive-contributions.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (hanzei) [reply site/content/contribute/getting-started/inactive-contributions.md]: That is correct. Would you mind including the changes in this PR?

File: site/content/contribute/getting-started/inactive-contributions.md (hunk 1 above)
Comment (Rajakavitha1) [reply site/content/contribute/getting-started/inactive-contributions.md]: Incorporated teh changes suggested. Please take a look. 
Note p

**Comment by hanzei** [comment]
Hey @Rajakavitha1,

I tried testing out your PR locally, but did run into problems with my hugo version. Unfortunately starting tomorrow I'm on vacation. Hence my review will be delayed. Sorry for this.

File: site/layouts/shortcodes/note.html [hunk 3]
```diff
@@ -2,5 +2,5 @@
     <div class="note-icon">
         {{partial "svg/exclamation.svg" (dict "size" "20px" ) }}
     </div>
-    <div class="note-content">{{- .Inner -}}</div>
+    <div class="note-content"><div><strong>Note:</strong></div>{{- .Inner -}}</div>
```
Comment (hanzei) [inline site/layouts/shortcodes/note.html]: Would you please remove this? I like the idea, but I would rather allow to configure this. See https://github.com/mattermost/mattermost-developer-documentation/pull/344/commits/902d50c634e99b23bb75b2b8cd0f3f623532c558

File: site/content/contribute/getting-started/inactive-contributions.md [hunk 4]
```diff
@@ -13,12 +13,13 @@ This process describes how inactive contributions are managed at Mattermost, ins
 
 2. After 20 days of inactivity, a contribution becomes inactive.
  - Community Coordinator asks the submitter if the team can help with questions. They acknowledge that after another 30 days of inactivity the contribution will be closed. They also add a `lifecycle/2:inactive` label to the contribution.
- - Note: Contributions should never become orphaned because of reviewers. The Coordinator will be responsible for receiving a response from the reviewers during the stale period, which may be that the maintainers aren't able to accept the contribution in its current form.
-
+  {{< note >}}
```
Comment (hanzei) [inline site/content/contribute/getting-started/inactive-contributions.md]: See below:
```suggestion
  {{< note "Note:" >}}
```

File: site/layouts/shortcodes/note.html (hunk 3 above)
Comment (Rajakavitha1) [reply site/layouts/shortcodes/note.html]: incorporated the changes

File: site/content/contribute/getting-started/inactive-contributions.md (hunk 4 above)
Comment (Rajakavitha1) [reply site/content/contribute/getting-started/inactive-contributions.md]: incorporated the changes suggested

**Pull request merged**